content/docs/command-reference/metrics/diff.md

```diff
@@ -61,8 +61,8 @@ all the current metrics (without comparisons).
   ```
 
 - `-R`, `--recursive` - determines the metrics files to use by searching each
-  target directory and its subdirectories for DVC-tracked files to inspect. If
-  there are no directories among the `targets`, this option is ignored.
+  target directory and its subdirectories for `dvc.yaml` files to inspect. If
```

Comment:
```suggestion
  target directory and its subdirectories for metrics files to inspect. If
```

`metrics` has different behavior because targets can be anything that can be parsed as metrics, even if not specified in `dvc.yaml`. For example, a JSON file with `{"acc": 0.9}` will be included as long as its somewhere in the target directory, even if no `dvc.yaml` exists.

Reply:
Oh true, good catch, updated! Is that also true for `metrics show`?

Reply:
>  Is that also true for metrics show -R?

I tested it on 2.9.3 and it seems so, so updated as well.

`metrics diff -R` isn't working as expected for me though... I'll open an issue ⌛

Reply:
> `metrics diff -R` isn't working as expected for me though...

Never mind, I forgot to provide an `a_rev` 😬